generators/client/templates/react/src/main/webapp/app/modules/administration/gateway/gateway.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Translate, ICrudGetAction } from 'react-jhipster';
+import { FaRefresh } from 'react-icons/lib/fa';
+
+import { getRoutes } from '../../../reducers/gateway';
+// import GatewayModal from './gateway-modal';
+
+export interface IGatewayPageProps {
+  getRoutes: ICrudGetAction;
+  routes: any[];
+  account: any;
+  isFetching?: boolean;
+}
+
+export class GatewayPage extends React.Component<IGatewayPageProps, any> {
```

Comment:
no need of second generic param for stateless componenet `React.Component<IGatewayPageProps>` is sufficient